src/pages/admin/categories/AddEditCategory/index.module.scss

```diff
@@ -1,19 +1,13 @@
 @use '../../../../styles/variables' as *;
 
+@use '../../../../styles/card-container.scss' as *;
```

Comment:
please rename the file to something that indicates it is for the general global styles and not just for card-container.

maybe something like 'global-styles'